### DISCUSSION TITLE
Xdrip Audio interference over bluetooth

### DISCUSSION THREAD

**Discussion by agentxcainy** [NightscoutFoundation/xDrip#2352] [Q&A]
Hi, I am using XDRIP with miaomiao and a Samsung fold 3. Quite often when playing music over bluetooth it will become garbled, usually after a reading update. Stopping the music and restarting usually fixes things. If I turn  xdrip off this doesn't happen. Has anyone else had this problem or have a solution please? Thanks, Mike

**Comment by Navid200** [comment]
What version of xDrip are you using?
You can see on the system status page.

Do you have Speak Readings enabled?

**Reply by agentxcainy** [reply]
![inbound7944633110697343184](https://user-images.githubusercontent.com/65816012/192164652-f7c22d3b-d470-420d-a6ff-b01e624120a0.jpg)

**Comment by agentxcainy** [comment]
Broadcast readings is not enabled. I recently changed to this version as I bought an amazfit bip watch but it used to do it with a previous version with a miband 5 watch

**Reply by Navid200** [reply]
There is no way we can support all forks from this repository.

If I were you, I would switch to an official xDrip release.  I would report the problem then.  
I cannot guarantee that would solve the problem.  But, I see no other way to move forward.
